helm/kanister-operator/values.yaml

```diff
@@ -36,6 +36,20 @@ controller:
     # false : kanister-prometheus framework has been disabled
     # true: kanister-prometheus framework has been enabled
     enabled: false
+  # Pod-level security context settings
+  podSecurityContext:
+    runAsUser: 1000
+    fsGroup: 2000
+    runAsNonRoot: true
+
+  # Container-level security context settings
+  containerSecurityContext:
+    capabilities:
+      drop:
+      - ALL
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    allowPrivilegeEscalation: false 
```

Comment:
I don't think this is required right? We just want to add the capability to configure the security context we don't necessarily want to run the containers/pods with specific security context at least in this PR.

Reply:
Ok, then i will comment out these field.

Reply:
I was just wondering why do we have to comment them? Can we not just remove them from here? This is what I wanted to request in my first comment in other discussion as well.

Reply:
got that. That also we can do.